sql_metadb/derived_tables/holding_notes.sql

```diff
@@ -0,0 +1,45 @@
+DROP TABLE IF EXISTS holdings_notes;
+CREATE TABLE IF EXISTS holdings_notes;

```

Comment:
I've done this before too :). The "If exists" needs to be removed along with the ";". Then add AS

Reply:
Please retain the IF EXISTS in DROP TABLE, but remove it from CREATE TABLE.